pkg/v3/tickers/schedule.go

```diff
@@ -0,0 +1,139 @@
+/*
+ScheduleTicker produces includes an added value in a tick SendDelay duration
```

Comment:
nit: typo. ScheduleTicker includes an added value?

Reply:
Updated the docs to make the behavior more clear.